core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/handler/ExportToOtherGraphHandler.java

```diff
@@ -46,97 +47,146 @@ protected OtherGraphExporter createExporter(final ExportToOtherGraph export, fin
     }
 
     protected Graph createGraph(final ExportToOtherGraph<?> export, final Store store) {
+        validationHandling(store, export);
 
         final String exportGraphId = export.getGraphId();
```

Comment:
I don't think we really need some of these variables. Then you can just pass the export operation to the methods instead of passing so many variables around.